test/release/examples/primers/fileIO.chpl

```diff
@@ -197,11 +199,12 @@ if example == 0 || example == 3 {
 
     // This is a forall loop to do I/O in parallel!
     forall i in 0..#num by -1 {
-
+      var start = 8*i;
+      var end = 8*i+8;
       // When we create the reader, supplying locking=false will do unlocked I/O.
       // That's fine as long as the channel is not shared between tasks;
       // here it's just used as a local variable, so we are O.K.
-      var r = f.reader(kind=ionative, locking=false, start=8*i, end=8*i+8);
+      var r = f.reader(kind=ionative, locking=false, region=start..end);
```

Comment:
Just looked at this while checking what example codes had changed in this release.  This seems like it would be an optimal opportunity for using `start..#8` rather than computing the `end` explicitly.

Reply:
Ah, good point!  I'll make a PR for that

Reply:
Looks like it needed to be `#9` instead, but otherwise that worked fine

Reply:
Oh, right...  Math!

Reply:
Opened #20670 but I think it's straight-forward enough to not need a review